example/haberdasher_connecpy.py

```diff
@@ -98,37 +99,37 @@ def MakeHat(
         self,
         request: haberdasher__pb2.Size,
         *,
-        ctx: Optional[ClientContext] = None,
+        headers: Optional[Headers] = None,
+        timeout_ms: Optional[int] = None,
         server_path_prefix: str = "",
         use_get: bool = False,
-        **kwargs,
     ) -> haberdasher__pb2.Hat:
         method = "GET" if use_get else "POST"
         return self._make_request(
             url=f"{server_path_prefix}/i2y.connecpy.example.Haberdasher/MakeHat",
-            ctx=ctx,
+            method=method,
+            headers=headers,
+            timeout_ms=timeout_ms,
             request=request,
             response_class=haberdasher__pb2.Hat,
-            method=method,
-            **kwargs,
         )
 
     def DoNothing(
         self,
         request: google_dot_protobuf_dot_empty__pb2.Empty,
         *,
-        ctx: Optional[ClientContext] = None,
+        headers: Optional[Headers] = None,
+        timeout_ms: Optional[int] = None,
         server_path_prefix: str = "",
-        **kwargs,
```

Comment:
`kwargs` makes it hard for users to know the actual arguments (they're whatever is passed to private `_make_request` method. So I switched to a pattern of explicit